ui/Screen/Project/Source.svelte

```diff
@@ -189,9 +188,7 @@
           <div class="repo-stat-item">
             <Icon.Commit />
             <Text style="margin: 0 8px;">
-              <a
-                href={path.projectCommits(project.id, $currentRevision)}
-                use:link>
+              <a href={`#${path.projectCommits(project.id, $currentRevision)}`}>
```

Comment:
We should add a comment about `link` not being able to handle reactivity, so this doesn't get refactored away accidentally.

Reply:
Actually, scrap that. Upgrading to the latest `svelte-spa-router` should fix this problem:
> You can now pass a parameter to the use:link action (e.g. use:link={val}) which will take precedence over the tag's href value. You can use this to create links whose target changes reactively, for example. (Thanks to @cianx!)

https://github.com/ItalyPaleAle/svelte-spa-router/releases

If you go for the upgrade, could also fix it in this place too:
https://github.com/radicle-dev/radicle-upstream/blob/master/ui/DesignSystem/Component/HorizontalMenu/MenuItem.svelte#L45

Reply:
Can do it in a separate ticket.